CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# Release v3.4.0
+
+- Support for Audio Streamer (lite) API via `stream_audio_to_websocket` method
```

Comment:
Please change "Audio Streamer (lite)" to "Audio Streamer" throughout.

Reply:
As discussed over slack, will rename to "Audio Connector"

Reply:
Have renamed throughout as requested